### PR TITLE
[DBZ-267] Enable images to run on OpenShift Online

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: kafka/0.5/Dockerfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
     rm -f /tmp/kafka.tgz
 
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
+RUN mkdir $KAFKA_HOME/config.template && mv $KAFKA_HOME/config/* $KAFKA_HOME/config.template
```
Comment (rhauch) [inline kafka/0.5/Dockerfile]: Can you explain the name `config.template`? Seems like it's more of a copy than templates.

Comment (jpechane) [inline kafka/0.5/Dockerfile]: It is a template for config dir, not templating engine

Comment (rhauch) [inline kafka/0.5/Dockerfile]: I read "template" and assumed it was for configuration files that were templates. May be worth considering a different name. WDYT?

Comment (jpechane) [inline kafka/0.5/Dockerfile]: How about .orig?

Comment (rhauch) [inline kafka/0.5/Dockerfile]: Sure, that is a bit clearer. Thanks!

**Comment by gunnarmorling** [comment]
Merged. Note that I've reworded the commit messages to contain the issue key. Thanks a lot, @jpechane!

**Pull request closed without merging**